mantle/cmd/kola/options.go

```diff
@@ -60,6 +60,7 @@ func init() {
 	ssv(&kola.BlacklistedTests, "blacklist-test", []string{}, "List of tests to blacklist")
 	bv(&kola.Options.SSHOnTestFailure, "ssh-on-test-failure", false, "SSH into a machine when tests fail")
 	sv(&kola.Options.CosaBuild, "cosa-build", "", "File path for coreos-assembler meta.json")
+	bv(&kola.Options.CosaBuildSchemaEnforce, "enforce-schema", true, "enforce schema validation of for meta.json")
```

Comment:
Shouldn't this just honor the existing `COSA_META_SCHEMA=none` environment variable that was added on the python side?

Reply:
It'd then probably be a 3-4 line patch instead of passing the bool all the way down the stack.

Reply:
Ha! Ironically, I looked for the best way do that, and thought this was the better path by having the cli front the pain. The `init` func in the `cosa` pkg is looking pretty hacky to me. But I'm fan for this.

Changed up, tested and repushed.